# generated by patch-package 6.4.8 on 2022-01-07 14:52:02
#
# command:
#   npx patch-package glob
#
# declared package:
#   glob: 7.2.0
#
diff --git a/node_modules/glob/glob.js b/node_modules/glob/glob.js
index afcf827..76649ca 100644
--- a/node_modules/glob/glob.js
+++ b/node_modules/glob/glob.js
@@ -42,7 +42,7 @@ module.exports = glob
 
 var rp = require('fs.realpath')
 var minimatch = require('minimatch')
-var Minimatch = minimatch.Minimatch
+// var Minimatch = minimatch.Minimatch
 var inherits = require('inherits')
 var EE = require('events').EventEmitter
 var path = require('path')
@@ -53,7 +53,7 @@ var common = require('./common.js')
 var setopts = common.setopts
 var ownProp = common.ownProp
 var inflight = require('inflight')
-var util = require('util')
+// var util = require('util')
 var childrenIgnored = common.childrenIgnored
 var isIgnored = common.isIgnored
 
diff --git a/node_modules/glob/sync.js b/node_modules/glob/sync.js
index 4f46f90..d4bb680 100644
--- a/node_modules/glob/sync.js
+++ b/node_modules/glob/sync.js
@@ -3,9 +3,9 @@ globSync.GlobSync = GlobSync
 
 var rp = require('fs.realpath')
 var minimatch = require('minimatch')
-var Minimatch = minimatch.Minimatch
-var Glob = require('./glob.js').Glob
-var util = require('util')
+// var Minimatch = minimatch.Minimatch
+// var Glob = require('./glob.js').Glob
+// var util = require('util')
 var path = require('path')
 var assert = require('assert')
 var isAbsolute = require('path-is-absolute')
